src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/Dumpling.targets

```diff
@@ -0,0 +1,37 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Setup Dumpling service to collect crash dumps -->
+  <Target Name="SetupDumpling"
+          BeforeTargets="RunTestsForProject"
+          Condition="'$(TargetOS)'!='Windows_NT'">
+    <ItemGroup>
+      <TestCommandLines Include="wget https://dumpling.azurewebsites.net/api/client/dumpling.py" />
```

Comment:
Any reason why you didn't just include this as a text file and call ReadLinesFromFile?  Seems like that might be more maintainable, IMO.